src/main/java/com/google/cloud/flink/bigquery/ArrowDeserializationSchema.java

```diff
@@ -113,6 +113,17 @@ public TypeInformation<T> getProducedType() {
     return (TypeInformation<T>) typeInfo;
   }
 
+  private Schema getSchema(String schemaJson) {
+    Schema schema = null;
+    try {
+      schema = Schema.fromJSON(schemaJson);
+    } catch (IOException e) {
+      logger.error("Error while converting to Schema from jsonString");
+      throw new FlinkBigQueryException("Error while converting to Schema from jsonString");
```

Comment:
please add the thrown exception as the cause of the new one: ` throw new FlinkBigQueryException("Error while converting to Schema from jsonString", e);`